Samples/outlook-check-item-categories/README.md

```diff
@@ -0,0 +1,225 @@
+
+1. Run the following command to generate a self-signed certificate to use for the web server.
+
+   ```console
+    npx office-addin-dev-certs install
+    ```
+
+    This command will display the folder location where it generated the certificate files.
+
+1. Go to the folder location where the certificate files were generated, then copy the **localhost.crt** and **localhost.key** files to the cloned or downloaded sample folder.
+
+1. Run the following command.
+
+    ```console
+    http-server -S -C localhost.crt -K localhost.key --cors . -p 3000
+    ```
+
+    The http-server will run and host the current folder's files on localhost:3000.
+
+1. Now that your localhost web server is running, you can sideload the **manifest-localhost.xml** file provided in the sample folder. Using this file, follow the steps in [Run the sample](#run-the-sample) to sideload and run the add-in.
+
+## Key parts of the sample
+
+### Configure event-based activation in the manifest
+
+The manifest configures a runtime to handle event-based activation. Because the Outlook platform uses the client to determine whether to use HTML or JavaScript to load the runtime, both of these files must be referenced in the manifest. An HTML page resource ID is specified in the `<Runtime>` element, and a JavaScript file resource ID is specified in the `<Override>` element, as shown below. Outlook on Windows uses the referenced JavaScript file, instead of the HTML page, to load the runtime.
+
+```xml
+<Runtime resid="WebViewRuntime.Url">
+    <Override type="javascript" resid="JSRuntime.Url"/>
+</Runtime>
+...
+<bt:Url id="WebViewRuntime.Url" DefaultValue="https://officedev.github.io/Office-Add-in-samples/Samples/outlook-check-item-categories/src/commands.html" />
+<bt:Url id="JSRuntime.Url" DefaultValue="https://officedev.github.io/Office-Add-in-samples/Samples/outlook-check-item-categories/src/commands/commands.js" />
+```
+
+The `<LaunchEvents>` element maps the four events that activate the add-in to the functions that handle each event. To determine how the add-in will handle the `OnMessageSend` and `OnAppointmentSend` events, an additional `SendMode` attribute is specified.
+
+```xml
+<LaunchEvents>
+    <LaunchEvent Type="OnNewMessageCompose" FunctionName="onItemComposeHandler"/>
+    <LaunchEvent Type="OnNewAppointmentOrganizer" FunctionName="onItemComposeHandler"/>
+    <LaunchEvent Type="OnMessageSend" FunctionName="onItemSendHandler" SendMode="SoftBlock"/>
+    <LaunchEvent Type="OnAppointmentSend" FunctionName="onItemSendHandler" SendMode="Block"/>
+</LaunchEvents>
+```
+
+The `OnMessageSend` event uses the `SoftBlock` option to prevent a user from sending a message if some or all of the required categories are missing. In the event that the add-in becomes unavailable (for example, if the add-in encounters a loading error) and can't check the message for applied categories, the user will still be able to send the message.
```

Comment:
```suggestion
The `OnMessageSend` event uses the `SoftBlock` option to prevent a user from sending a message if any of the required categories are missing. In the event that the add-in becomes unavailable (for example, if the add-in encounters a loading error) and can't check the message for applied categories, the user will still be able to send the message.
```